src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -3479,4 +3480,42 @@ WindowID MessageHelper::ExtractWindowIdFromSmartObject(
   return mobile_apis::PredefinedWindows::DEFAULT_WINDOW;
 }
 
+void MessageHelper::AddDefaultParamsToTireStatus(
+    ApplicationSharedPtr app, smart_objects::SmartObject& response_from_hmi) {
+  const utils::SemanticVersion max_version_with_mandatory_params(7, 1, 0);
+
+  if (!app) {
+    SDL_LOG_ERROR("Application not found");
+    return;
+  }
+
+  if (app->msg_version() > max_version_with_mandatory_params) {
+    SDL_LOG_DEBUG(
+        "Tire status parameters are "
+        "non-mandatory for this app version, no need in default values");
+    return;
+  }
+
+  smart_objects::SmartObject& tire_status =
+      response_from_hmi[strings::msg_params][strings::tire_pressure];
+
+  if (!tire_status.keyExists(strings::pressure_telltale)) {
+    tire_status[strings::pressure_telltale] =
+        mobile_apis::WarningLightStatus::WLS_NOT_USED;
+  }
+
+  const std::vector<std::string> tires{strings::left_front,
+                                       strings::right_front,
+                                       strings::left_rear,
+                                       strings::right_rear,
+                                       strings::inner_left_rear,
+                                       strings::inner_right_rear};
+  for (const std::string& tire : tires) {
```

Comment:
Is it possible to include previously received/cached value? Only if that is already available.

Reply:
@atiwari9 As I see for now there is no possibility to include previous values because there is no place to store them. Subscription data is stored into VehicleInfoAppExtension, but execution results of GetVehicleData requests or OnVehicleData notifications aren't cached in any place.

Reply:
Ok understood, Can we please include this in some place like HMI guidelines or include this information in Info param for GVD response?

Reply:
@atiwari9, please clarify, do you want to add 
> Note: SDL doesn't cash execution results of GetVehicleData requests 

to
https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/develop/docs/VehicleInfo/GetVehicleData/index.md 

and

> Note: SDL doesn't cash execution results of OnVehicleData notifications

to
https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/develop/docs/VehicleInfo/OnVehicleData/index.md

Reply:
I would want to clarify that for msg version earlier to 8.0.0, the `leftFront`, `leftRear` etc params may contain default(not-current) values if HMI does not send those.

Reply:
Tracking in `HMI_GUIDELINES` : https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/231#pullrequestreview-691708054